### ISSUE TITLE
run_iv cool_wait at a voltage variable value instead of the highest bias_voltage

### ISSUE THREAD

**Issue by yuhanwyhan** [slaclab/pysmurf#563]
## Describe the problem
The current run_iv has cool_wait at np.max(bias).

## Describe the solution you'd like
 Can there be a variable like coolbias_voltage, which if None things do the default thing and if a value, then waits there instead?

## Additional details
Waiting at different value (lower than the highest bias voltage), can help reduce the heat generated during the cool time

**Comment by eyyoung24** [comment]
@yuhanwyhan Do you know how to make this change? Happy to help.

**Comment by yuhanwyhan** [comment]
Yes, should I just modify it and submit a pull request later? @eyyoung24

**Comment by eyyoung24** [comment]
Yup. That's right @yuhanwyhan Maybe message me on Slack when it's ready to merge, I'll see the message sooner.
